proto/spacemesh/mesh.proto

```diff
@@ -20,9 +21,21 @@ service MeshService {
     // Number of layers per epoch (a network parameter)
     rpc EpochNumLayers(google.protobuf.Empty) returns (SimpleInt);
 
+    ////////// Methods designed for light-weight clients such as wallets
+    ////////// Don't use in clients such as dash, explore or backup
+    ////////// Use streams for these
+
+    // Get transactions to an account or from an account
+    rpc AccountTransactions(AccountIdWithOffset) returns (Transactions);
+
     ////////// Streams
 
-    // Layer with blocks and transactions.
-    // Sent each time layer data changes. Designed for heavy-duty clients.
+    // A stream of new txs (or update to a tx data) where the account is sender
+    // or receiver. These should include activation transations (ATXs) and all
+    // transactions in the mesh
+    rpc AccountTransactionStream(AccountId) returns (stream Transaction);
```

Comment:
As I commented above - this will be extremely wasteful in terms of number of messages that are sent and needs to be processed by clients. Wallets are only interested in a very small subset of accounts and not in every transaction in the network. Without this - clients will have compare all the accounts they care about with each transaction in the network - constant processing instead of just getting the data they need and acting upon it...

Reply:
#26